frontend/public/locales/en/translation.json

```diff
@@ -371,10 +374,7 @@
     "getTheExtensionButton": "Get the extension",
     "extensionNotAvailableOnYourBrowser": "The extension is not available on your web browser. You may use it on <2>Firefox</2>, <4>Google Chrome</4> or <6>Chromium</6>.",
     "collaborativeContentRecommendations": "Collaborative Content Recommendations",
-    "tournesolPlatformDescription": "Tournesol is an <1>open source</1> platform which proposes a tool for collaborative decision. The main aim of this tool is to <3>collaboratively</3> identify top videos of public utility by eliciting contributors' judgements on content quality. Currently we are also exploring the possibility of using Tournesol's algorithms to evaluates candidates in an election.<br><br>Thanks to the data collected on Tournesol, we hope to contribute to making today's and tomorrow's large-scale algorithms <6>robustly beneficial</6> for all of humanity.",
-    "videos": {
-      "start": "Start" 
```

Comment:
Just to explain the reason of these 3 lines: it's not used because it was supposed to be the home button for the tutorial. But as long as it is not used and we have in mind that uncommenting the tuto button of the home page will not be enough to make it functional, I have no objection to delete these unused lines

Reply:
Hello @adrausch 

That's true. In fact those lines have been automatically deleted by `yarn i18n:parse` because the parser found no non-commented occurrence of the translation key `videos.start` in the code. They will be added automatically again when we will enable the video tutorial : )